src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -27,6 +36,7 @@ public class CommandBox extends UiPart<Region> {
     public CommandBox(CommandExecutor commandExecutor) {
```

Comment:
Good use of Dependency Injection using Constructor Injection here to reduce the coupling